Common/Thread/ParallelLoop.cpp

```diff
@@ -106,6 +106,7 @@ void ParallelRangeLoop(ThreadManager *threadMan, const std::function<void(int, i
 	// and process the remainder right here. If there's no remainer, we'll steal a whole chunk.
 	if (counter) {
 		counter->WaitAndRelease();
+		delete counter;
```

Comment:
WaitAndRelease performs the delete already, so this is not needed (and would be a double-free).

Reply:
fix it https://github.com/hrydgard/ppsspp/pull/18967/commits/9c3c23daa6d3a70921f3aeb3560c81b497065eb4